torch_xla_py/xla_dist.py

```diff
@@ -402,3 +409,363 @@ def get_cluster(self):
     cluster.validate()
     return cluster
 
+
+class DistributedExecutor(object):
+
+  SCRIPT_PATH_TMPL = '/tmp/{pid}/dist_training_ptxla_{worker}.sh'
+  MASTER_IDX = 0
+  MESH_SERVICE_PORT = 8477  # Use single port to disallow concurrent runs
+  DIST_ENV_VARS = [
+      'XRT_TPU_CONFIG', 'XRT_LOCAL_WORKER', 'XRT_MESH_SERVICE_ADDRESS'
+  ]
+  DEFAULT_CONTAINER_NAME = 'pytorchtpudistrunner'
+  DEFAULT_USER_NAME = 'pytorchtpudistrunner'
+
+  @staticmethod
```

Comment:
In Google `@staticmethod` is not suggested. So I think this does not need to be a class method, but can be a normal function/API.
No need to the `_` prefix either.

Reply:
Done and moved out.